### DISCUSSION TITLE
Using internal stylesheets for CSS modules

### DISCUSSION THREAD

**Discussion by jziegel** [remix-run/remix#6515] [Q&A]
I've set up [CSS Bundling](https://remix.run/docs/en/main/guides/styling#css-bundling) and [CSS Modules](https://remix.run/docs/en/main/guides/styling#css-bundling) on a project, which creates a CSS file with combined styles from CSS modules, which can then be loaded via the `LinksFunction`. 

Is there any way we can set the combined styles on an internal stylesheet? This would be useful for some purposes, e.g. generating email templates which cannot load external stylesheets.

**Comment by cliffordfajardo** [comment]
Not sure I understood the second part of your question:
> Is there any way we can set the combined styles on an internal stylesheet? 

Are you asking if there is a way to have the generated CSS  be inlined all in a  `<style>` tag?
CC @sergiodxa or @markdalgleish - do you have any recommendations? 🙏
